src/manager/EzyAppManager.cpp

```diff
@@ -5,6 +5,23 @@
 
 EZY_NAMESPACE_START_WITH(manager)
 
+using namespace std;
+
+#if (cplusplus <= 201703L)
+namespace stdex {
+    template<class InputIt, class T>
+    constexpr InputIt find(InputIt first, InputIt last, const T& value)
+    {
+        for (; first != last; ++first) {
```

Comment:
Em tham khảo thì mình không nên gán lại giá trị của paramter: https://ncrcoe.gitbooks.io/java-for-small-teams/content/specifics/200_do_not_reassign_parameters.html
Em nghĩ tốt hơn sẽ là:

```cpp
namespace stdex {
template<class InputIt, class T>
constexpr InputIt find(InputIt first, InputIt last, const T& value){
    for (InputIt answer = first; answer != last; ++answer) {
        if (*answer == value) {
            return first;
        }
    }
    return last;
}
}
```

anh thấy sao ạ?

Reply:
Anh thấy dùng biến Iter mới sẽ trong sáng/rõ ràng hơn, đẻ anh fix lại chỗ này theo suggest của em